compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/schema/generator/GraphqlSourceModifier.java

```diff
@@ -146,6 +148,23 @@ private ModulePartNode modifyDocument(SourceModifierContext context, ModulePartN
                     nodeMap.put(targetNode, updatedNode);
                     this.entityTypeNamesMap.put(targetNode, this.entityUnionTypeName);
                     this.entityUnionSuffix++;
+                } else if (targetNode.kind() == SyntaxKind.OBJECT_CONSTRUCTOR) {
+                    ObjectConstructorExpressionNode graphqlServiceVariableDeclaration
+                            = (ObjectConstructorExpressionNode) targetNode;
+                    ObjectConstructorExpressionNode updatedGraphqlServiceObject = modifyServiceObjectNode(
+                            graphqlServiceVariableDeclaration, schemaString, prefix);
+                    VariableDeclarationNode parentNode
+                            = (VariableDeclarationNode) graphqlServiceVariableDeclaration.parent();
+                    VariableDeclarationNode updatedParentNode = parentNode.modify()
+                                                                          .withInitializer(updatedGraphqlServiceObject)
+                                                                          .apply();
+                    Node replacingNode = updatedParentNode;
+                    NonTerminalNode iNode = parentNode;
```

Comment:
`iNode` does not explain the purpose of the variable correctly. Shall we use a better name?